apm-sniffer/apm-agent-core/pom.xml

```diff
@@ -126,6 +131,12 @@
                 <artifactId>guava</artifactId>
                 <version>${guava.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${lombok.version}</version>
+                <scope>provided</scope>
```

Comment:
@wu-sheng The scope of lombok dependency is provided. It won't be packaged into agent library. So please don't worry about the dependency will affect agent core.

Reply:
My points are two
1. Version should be managed at root pom, as Lombok is used at backend and agent.
1. With this feature added, a lot of codes could be removed at the agent side. It should be placed in another PR, with more codes changed. This will make the changelog clear when I do release several months later.

Reply:
Copy that. I will revert all changes related `lombok`.

Reply:
Submit an issue If Any of you want to add this later.

Reply:
Clear. 

BTY, I have reverted all changes related `lombok`, waiting for checking.